components/abtesting/optimizelyXExperiment/README.md

```diff
@@ -41,8 +41,113 @@ Component will obtain OptimizelyX's decision for given `experimentId` and displa
 
 If no variation is provided or OptimizelyX API is unavailable, `defaultVariation` will be displayed.
 
-## Known issues
+## Advanced Usage
+
+### Experiment Context
+
+The following data is set in a plain object to the experiment context.
+
+- All props passed to OptimizelyXExperiment (see limitations at Known issues section).
+- isActive: true when a decision has been made by optimizely.
```

Comment:
A suggestion to improve the README. You could use code styling for values that improves the readability of those.

```suggestion
- isActive: `true` when a decision has been made by optimizely.
```